$OpenBSD: patch-main-menu_src_main-menu-ui_c,v 1.1.1.1 2010/10/25 17:43:15 ajacoutot Exp $

From 6532ab1edc544ddc40e2a012500aa457b3920a4d Mon Sep 17 00:00:00 2001
From: Julian Andres Klode <jak@debian.org>
Date: Tue, 15 Jun 2010 11:38:07 +0000
Subject: [main-menu] Use gtk_widget_get_visible instead of GTK_WIDGET_VISIBLE

Use the gtk_widget_get_visible() function instead of the deprecated
GTK_WIDGET_VISIBLE macro.

--- main-menu/src/main-menu-ui.c.orig	Fri May 21 12:46:08 2010
+++ main-menu/src/main-menu-ui.c	Mon Oct 25 11:04:32 2010
@@ -1339,7 +1339,7 @@ update_limits (MainMenuUI *this)
 	priv->max_total_items = max_total_items_default;
 
 	for (i = 0; i < 2; ++i) {
-		if (GTK_WIDGET_VISIBLE (sections [i]))
+		if (gtk_widget_get_visible (sections [i]))
 			g_object_get (
 				user_tables [i],
 				"n-rows", & n_rows, "n-columns", & n_cols, NULL);
@@ -1458,22 +1458,22 @@ set_table_section_visible (MainMenuUI *this, TileTable
 		gtk_widget_hide (priv->table_sections [table_id]);
 
 	if (
-		GTK_WIDGET_VISIBLE (priv->table_sections [USER_APPS_TABLE]) ||
-		GTK_WIDGET_VISIBLE (priv->table_sections [RCNT_APPS_TABLE])
+		gtk_widget_get_visible (priv->table_sections [USER_APPS_TABLE]) ||
+		gtk_widget_get_visible (priv->table_sections [RCNT_APPS_TABLE])
 	)
 		gtk_widget_show (priv->page_selectors [APPS_PAGE]);
 	else
 		gtk_widget_hide (priv->page_selectors [APPS_PAGE]);
 
 	if (
-		GTK_WIDGET_VISIBLE (priv->table_sections [USER_DOCS_TABLE]) ||
-		GTK_WIDGET_VISIBLE (priv->table_sections [RCNT_DOCS_TABLE])
+		gtk_widget_get_visible (priv->table_sections [USER_DOCS_TABLE]) ||
+		gtk_widget_get_visible (priv->table_sections [RCNT_DOCS_TABLE])
 	)
 		gtk_widget_show (priv->page_selectors [DOCS_PAGE]);
 	else
 		gtk_widget_hide (priv->page_selectors [DOCS_PAGE]);
 
-	if (GTK_WIDGET_VISIBLE (priv->table_sections [USER_DIRS_TABLE]))
+	if (gtk_widget_get_visible (priv->table_sections [USER_DIRS_TABLE]))
 		gtk_widget_show (priv->page_selectors [DIRS_PAGE]);
 	else
 		gtk_widget_hide (priv->page_selectors [DIRS_PAGE]);
@@ -1887,7 +1887,7 @@ panel_button_clicked_cb (GtkButton *button, gpointer u
 	detector = DOUBLE_CLICK_DETECTOR (
 		g_object_get_data (G_OBJECT (toggle), "double-click-detector"));
 
-	visible = GTK_WIDGET_VISIBLE (priv->slab_window);
+	visible = gtk_widget_get_visible (priv->slab_window);
 
 	if (! double_click_detector_is_double_click (detector, gtk_get_current_event_time (), TRUE)) {
 		if (! visible)
@@ -1895,7 +1895,7 @@ panel_button_clicked_cb (GtkButton *button, gpointer u
 		else
 			gtk_widget_hide (priv->slab_window);
 
-  		visible = GTK_WIDGET_VISIBLE (priv->slab_window);
+		visible = gtk_widget_get_visible (priv->slab_window);
 	}
 
 	gtk_toggle_button_set_active (priv->panel_button, visible);
